docs/plugins.yml

```diff
@@ -24,6 +24,7 @@
 - raintank/snap-plugin-collector-memcache
 - raintank/snap-plugin-collector-snapstats
 - raintank/snap-plugin-collector-tcpconns
+- raintank/snap-plugin-collector-kubestate
```

Comment:
Thank you for this plugin, but I think that we already have kubestate collector on the list - line 21...